src/main/java/com/kodilla/ecommercee/repository/UserRepository.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.users.User;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+@Transactional
```

Comment:
`@Transactional` nie jest tu potrzebne. Ogólnie nie zalecam używania tej annotacji nad interfejsami. To czy dana implementacja ma wspierać transakcję jest własnością serwisu implementującego interfejs, a nie samego interfejsu. Dodatkowo implementacja CrudRepository (`SimpleJpaRepository`) ma w sobie już oznaczenie `@Transactional`
 @DonnieDarko8881 @SlawomirJablonski Rozpisywałem się z którymś z Was o tym ;) Możecie podyskutować o tym.

Reply:
pieknie dziekuje za wytlumaczenie